discovery-provider/alembic/versions/38642fb2948d_add_pg_triggers.py

```diff
@@ -26,6 +26,10 @@ def upgrade():
     connection = op.get_bind()
     connection.execute(
         """
+        SELECT pg_cancel_backend(pid)
+        FROM pg_stat_activity 
+        WHERE state != 'idle' AND query NOT ILIKE '%pg_stat_activity%';
```

Comment:
is that right? I thought `!=` in sql was `<>`

Reply:
https://stackoverflow.com/a/24060451